dependencies.yaml

```
@@ -29,43 +33,64 @@ files:
     includes:
       - cuda_version
       - py_version
-      - test_python
+      - test_python_ucxx
+      - test_python_distributed_ucxx
       - depends_on_cupy
       - depends_on_cudf
   checks:
     output: none
     includes:
       - checks
       - py_version
-  docs:
-    output: none
-    includes:
-      - docs
-      - py_version
-  py_build:
+  py_build_ucxx:
```

Accidentally removed docs key.
```suggestion
  docs:
    output: none
    includes:
      - docs
      - py_version
  py_build_ucxx:
```